DemoqaProject/pageObjects/Elements.cs

```diff
@@ -7,10 +7,323 @@ namespace DemoqaProject.PageObjects
     public class Elements : BaseSeleniumPage
     {
 
+        [FindsBy(How = How.XPath, Using = "//span[text()='Text Box']")]
+        public IWebElement textBoxButton;
+
+        [FindsBy(How = How.XPath, Using = "//input[@id='userName']")]
```

Comment:
Instead of How.XPath you can use simple How.Id I suppose. XPath is pretty slow and if there is a way to use name, id, css is better to use them instead of XPath.

Reply:
It's up to you update it or not, but as simple it'll be the better.

Reply:
done